tests/src/system/basic/WskSdkTests.scala

```diff
@@ -42,18 +45,29 @@ class WskSdkTests
         val dir = File.createTempFile("wskinstall", ".tmp")
         dir.delete()
         dir.mkdir() should be(true)
-
-        wsk.cli(wskprops.overrides ++ Seq("sdk", "install", "docker"), workingDir = dir).
-            stdout should include("The docker skeleton is now installed at the current directory.")
-
-        val sdk = new File(dir, "dockerSkeleton")
-        sdk.exists() should be(true)
-        sdk.isDirectory() should be(true)
-
-        val file = new File(sdk, "Dockerfile")
-        file.exists() should be(true)
-        file.isFile() should be(true)
-        FileUtils.deleteDirectory(dir)
+        try {
+            wsk.cli(wskprops.overrides ++ Seq("sdk", "install", "docker"), workingDir = dir).
+                stdout should include("The docker skeleton is now installed at the current directory.")
+
+            val sdk = new File(dir, "dockerSkeleton")
+            sdk.exists() should be(true)
+            sdk.isDirectory() should be(true)
+
+            val dockerfile = new File(sdk, "Dockerfile")
+            dockerfile.exists() should be(true)
+            dockerfile.isFile() should be(true)
+            val lines = FileUtils.readLines(dockerfile)
```

Comment:
Any particular reason why you use Apache Commons vs. `Source.fromPath("myfile.txt").getLines()`?